### PR TITLE
test 123 fix

### PR DESCRIPTION
**Pull request by chackohead**

### CODE REVIEW

**Comment by chackohead** [comment]
Looks good!

**Comment by takazumu** [comment]
looks good

**Review by takazumu — COMMENTED** [review]
looking good durng the code review

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
...
+Super good stuff
+
 This is where the notes for this repo exist.
-adding more comments
- haveing fun
- 
\ No newline at end of file
+adding more comments haveing fun
+ 
+Some more read me updates from me - tzumu.
```
Comment (chackohead) [inline README.md:8]: remove your name

File: README.md [hunk 2]
```diff
@@ -1,5 +1,8 @@
 # codingFun
+
+Super good stuff
+
 This is where the notes for this repo exist.
-adding more comments
- haveing fun
- 
\ No newline at end of file
+adding more comments haveing fun
```
Comment (chackohead) [inline README.md:6]: fix mispelling of having

**Pull request merged**